en/docs/design/api-policies/attach-policy.md

```diff
@@ -31,53 +51,27 @@ Follow the instructions below to attach one or more default policies that are sh
     </table>
 
     !!! note 
-        You can optionally use the **Apply to all** resources option to attach the same policy to all the resources on Save. This will attach the same policy to all the API operations along with the values you entered to configure the policy (if any). It is important to note that if the policy was applied to the Request Flow, it will only be applied to all the Request Flows of all operations.
+        You can optionally use the **Apply to all** resources option to attach the same policy to all the resources on Save. This will attach the same policy to all the API operations along with the values you entered to configure the policy (if any). It is important to note that if the policy was applied to the `Request Flow`, it will only be applied to all the `Request Flows` of all operations.
```

Comment:
```suggestion
        You can optionally use the **Apply to all** resources option to attach the same policy to all the resources when you save your settings. This will attach the same policy to all the API operations along with the values you entered to configure the policy (if any). It is important to note that if the policy was applied to the `Request Flow`, it will only be applied to all the `Request Flows` of all operations.
```